docs/user/alerting/alerting-scale-performance.asciidoc

```diff
@@ -0,0 +1,28 @@
+[role="xpack"]
+[[alerting-scale-performance]]
+== Scale and performance
+
+{kib} alerting run both alert checks and actions as persistent background tasks. This has two major benefits:
```

Comment:
How about:

Alerting runs alert checks and actions as persistent background tasks.